conjure-api/src/main/conjure/conjure-api.yml

```diff
@@ -158,6 +158,7 @@ types:
           docs: optional<Documentation>
           deprecated: optional<Documentation>
           markers: list<Type>
+          tags: set<string>
```

Comment:
I'm just realizing this is on the endpoint and not the arguments. If this is meant to replace the `Safe` marker, don't we need it on every arg?

Reply:
Yes, we should add the tags field to parameters as well. Good catch